lib/omnibus-flapjack/publish.rb

```diff
@@ -198,6 +199,28 @@ def add_to_rpm_repo(pkg, component = 'experimental')
           Mixlib::ShellOut.new('createrepo . 2>&1', :live_stream => $stdout).run_command.error!
         end
       end
+
+      def add_to_packagecloud(pkg, component='experimental')
+        begin
+          credentials = Packagecloud::Credentials.new(ENV['packagecloud_user'], ENV['packagecloud_token'])
```

Comment:
Yep, good thought.  I'll merge this now, and as discussed, move all the env variables to one place in the Rakefile.